### PR TITLE
Fix #15850: Display content from information_schema.PROCESSLIST

### LINKED ISSUE #15850: Cant display content from information_schema.PROCESSLIST
### Describe the bug

Display nothing. Also, if you take manually select from this table, you're got that error:
```
Fatal error: Uncaught TypeError: trim() expects parameter 1 to be string, null given in /usr/share/phpMyAdmin/libraries/classes/Display/Results.php:2864 Stack trace: #0 /usr/share/phpMyAdmin/libraries/classes/Display/Results.php(2864): trim(NULL) #1 /usr/share/phpMyAdmin/libraries/classes/Display/Results.php(2614): PhpMyAdmin\Display\Results->_getRowValues(Object(mysqli_result), Array, 0, false, Array, '', false, 'select * from i...', Array) #2 /usr/share/phpMyAdmin/libraries/classes/Display/Results.php(4300): PhpMyAdmin\Display\Results->_getTableBody(Object(mysqli_result), Array, Array, Array, false) #3 /usr/share/phpMyAdmin/libraries/classes/Sql.php(1666): PhpMyAdmin\Display\Results->getTable(Object(mysqli_result), Array, Array, false) #4 /usr/share/phpMyAdmin/libraries/classes/Sql.php(2002): PhpMyAdmin\Sql->getHtmlForSqlQueryResultsTable(Object(PhpMyAdmin\Display\Results), './themes/pmahom...', NULL, Array, false, false, 25, NULL, Object(mysqli_result), Array) #5 /usr/share/phpMyAdmin/libraries/classes/Sql.php(2271): P in /usr/share/phpMyAdmin/libraries/classes/Display/Results.php on line 2864
```

### To Reproduce

1. Go to SQL
2. execute `select * from information_schema.PROCESSLIST`
3. show nothing

### Expected behavior
Should shows process list

### Server configuration

CentOS, PHP 7.4, PMA 5.0.1

### PR DESCRIPTION
**Pull request by rabeehrz**
Signed-off-by: Mohammed Rabeeh <mohammadrabeeh@gmail.com>

### Description

Fixes #15850 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/Display/Results.php [hunk 1]
```diff
@@ -2859,6 +2859,7 @@ private function _getRowValues(
             $tblLower = mb_strtolower($meta->orgtable);
             $nameLower = mb_strtolower($meta->orgname);
             if (! empty($this->transformation_info[$dbLower][$tblLower][$nameLower])
+                && ($row[$i])
```
Comment (williamdes) [inline libraries/classes/Display/Results.php]: Can you use a function for example is_null or isset?

**Review by williamdes — CHANGES_REQUESTED** [review]

File: libraries/classes/Display/Results.php (hunk 1 above)
Comment (rabeehrz) [reply libraries/classes/Display/Results.php]: @williamdes Change made as requested. Please review.

**Review by williamdes — APPROVED** [review]
LGTM

**Comment by williamdes** [comment]
Thank you for your contribution @rabeehrz !
:1st_place_medal:

**Pull request merged**